charts/incubator/alfresco-infrastructure/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart Providing the required Alfresco infrastructure. This includes nginx-ingress, activemq, alfresco-api-gateway, alfresco-keycloak, and the content services volume.
 name: alfresco-infrastructure
-version: 0.1.0
+version: 0.1.0-DEPLOY-426
```

Comment:
Is this done for testing ?

Reply:
Yes, otherwise we cannot get a green build since there are 3 helm charts being modified.